### ISSUE TITLE
Page doesn't scroll up for the very bottom buttons automatically (build 58)

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-alert-app#800]
With VoiceOver on, when the user is at the bottom of these two onboarding screens (Step 3 and Step 5), the page doesn't automatically scroll up to show these clearly. Since there is nothing after this text, the next item is "Back" button. The text visible on the screen ends up with failed colour contrast since it's blurred/greyed out (cause it's at the end of the text area).

Please see screen shots.

iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/87974912-d07e4f80-ca98-11ea-9d0c-f0fa6f5ad30e.png)

![image](https://user-images.githubusercontent.com/14280062/87974880-bd6b7f80-ca98-11ea-884b-196c705571ec.png)

**Comment by JuliannaR** [comment]
@qanitah this is a visual fade and when the screen continues to scroll downward the colour of this changes to be dark and readable. When the element is focused it meets contrast. This is a visual cue that there is content below the fold.

**Comment by qanitah** [comment]
Yes it's the fade for the bottom of the screen but page doesn't automatically scroll up. The next item is Back button. It never becomes actual colours with VoiceOver on since there's nothing below this. It's also for the Provinces screen (Step 6).

**Comment by timarney** [comment]
Can the bottom of these pages have some additional padding added to help with the issue?

**Comment by JuliannaR** [comment]
@timarney it would be a workaround but would help.

**Comment by timarney** [comment]
Closing this needs to be retested on latest versions.
